gui/default/vendor/webauthn-json-1.0.3/webauthn-json.browser-global.js

```diff
@@ -0,0 +1,232 @@
+(() => {
```

Comment:
@emlun This PR is truly amazing work, and I don't want to delay its release by asking, but why include version 1.0.3, and not the latest 2.1.1 release? Also, the 2.x branch [recommends](https://github.com/github/webauthn-json?tab=readme-ov-file#api-browser-ponyfill) using the new ponyfill logic. Does that recommendation apply to us in this context? Again, I am not suggesting any changes be made, I am just asking us to note why we are pinning on 1.0.3.

Reply:
It's just that 1.0.3 was the latest non-alpha version back when I started this - [1.0.3 released 2022-06-20](https://github.com/github/webauthn-json/releases/tag/v1.0.3), I [pinned it on 2022-07-02](https://github.com/syncthing/syncthing/pull/8417/commits/a9c1304aedb296f0583338059d21b0cecea87642) and [2.0.0 released 2022-07-19](https://github.com/github/webauthn-json/releases/tag/v2.0.0).

We can probably upgrade it with minimal effort, I can take a look at that.

Reply:
Looks like the ponyfill is only available as an ES module, but the previous API works fine without any changes other than upgrading the vendored lib ([looks like it's almost entirely whitespace changes](https://github.com/syncthing/syncthing/pull/9175/commits/8f1ae2372a8dffc4c184224fd038a62755476c2b#diff-0098f688660357840b624cbec1c7c7b3ac9fffeb273af6ca7c6473a790c22f41)). I've pushed the update.